lib/index.js

```diff
@@ -15,329 +15,303 @@ const internals = {};
 
 exports.genMigrationFile = (options, cb) => {
 
-    Joi.assert(options, Schema.options, 'Bad options passed to schwifty-migration.');
+    const validOptions = Joi.validate(options, Schema.options);
+    if (validOptions.error) {
+        return cb(new Error(`Bad options passed to schwifty-migration: ${validOptions.error.message}`));
+    }
 
     const userOptions = Hoek.shallow(options);
 
     if (userOptions.models.length === 0) {
-        return cb(null, '');
+        return cb(null, 'No models passed');
```

Comment:
Should we be returning some sort of structured object or constant to indicate what happened?

Reply:
This changed to an error